client/src/components/ContactUs/Form.tsx

```diff
@@ -1,99 +1,106 @@
-                  </label>
-                  <input
-                    type="text"
-                    id="name"
-                    name="name"
-                    className="w-full bg-gray-100 rounded border border-gray-300 focus:border-blue-400 text-base outline-none text-gray-700 py-1 px-3 leading-8 transition-colors duration-200 ease-in-out"
-                    onChange={(e) => setName(e.target.value)}
-                    required
-                  />
-                </div>
-              </div>
-              <div className="p-2 w-1/2">
-                <div className="relative">
-                  <label className="leading-7 text-sm text-gray-600">
-                    Email
-                  </label>
-                  <input
-                    type="email"
-                    id="email"
-                    name="email"
-                    className="w-full bg-gray-100 rounded border border-gray-300 focus:border-blue-400 text-base outline-none text-gray-700 py-1 px-3 leading-8 transition-colors duration-200 ease-in-out"
-                    onChange={(e) => setEmail(e.target.value)}
-                    required
-                  />
-                </div>
-              </div>
-              <div className="p-2 w-full">
-                <div className="relative">
-                  <label className="leading-7 text-sm text-gray-600">
-                    Message
-                  </label>
-                  <textarea
-                    id="message"
-                    name="message"
-                    className="w-full bg-gray-100 rounded border border-gray-300 focus:border-blue-400 h-32 text-base outline-none text-gray-700 py-1 px-3 resize-none leading-6 transition-colors duration-200 ease-in-out"
-                    onChange={(e) => setMessage(e.target.value)}
-                    required
-                  ></textarea>
-                </div>
-              </div>
-              <div className="p-2 w-full">
-                <button className="flex mx-auto text-white bg-blue-400 border-0 py-2 px-8 focus:outline-none hover:bg-blue-600 rounded text-lg">
-                  Submit
-                </button>
-              </div>
-            </div>
-          </div>
-        </div>
-      </form>
-      <ToastContainer />
-    </div>
-  );
-};
-
-export default Form;
+import { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import axios from 'axios';
+import { ToastContainer, toast } from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
+import emailjs from 'emailjs-com';
+
+const Form = () => {
+  const [name, setName] = useState<String>('');
+  const [email, setEmail] = useState<String>('');
+  const [message, setMessage] = useState<String>('');
+
+  const history = useHistory();
+
+  const formSubmitHandler = (e: React.FormEvent) => {
+    e.preventDefault();
+    emailjs.sendForm('service_ohl97gj', 'template_hos8tvj','#contactForm', '-adufJCD2tG24TRJJ')

```

Comment:
This block of code should go under the then() of Axios request. Once the data is stored in the DB then only we start mailing people regarding it.